pymongo_inmemory/downloader.py

```diff
@@ -137,7 +192,7 @@ def _copy_bins():
         logger.debug("Copied {}".format(binfile_name))
 
 
-def _download_tar(dl_url, tar_file, dst_file):
+def _download_file(dl_url, dled_file, dst_file):
```

Comment:
Whats the purpose of `dled_file` ? doesnt look like its used in this function, which downloads the file to `dst_file`.
Wast trying to figure out but the main download function seems to define both `dled_file` and `dst_file` in the same way:
```
dled_file = os.path.join(dl_folder, dled_file_pattern.format(ver=version))    
dst_file = os.path.join(dl_folder, dled_file_pattern.format(ver=version))
```

Reply:
It might have been an oversight, i.e. was being used at some point but forgot to remove it. I'll have a look.